roles/openshift_master/README.md

```diff
@@ -1,7 +1,7 @@
-OpenShift Master
+Cluster Master
```

Comment:
This name references the role name, which is still openshift_master...  Renaming this here would affect publishing this playbook to Ansible Galaxy...  As long as the role stays openshift_master, this readme should still reference that name.